common-pipelines.yml

```diff
@@ -0,0 +1,21 @@
+jobs:
+  - job: ${{ parameters.name }}
+    pool:
+      vmImage: ${{ parameters.vmImage }}
+
+    steps:
+      - task: UsePythonVersion@0
+        displayName: "Use Python 3.6"
```

Comment:
only for py3.6? how about py2.7?

Reply:
The vm image of these three platform has pre-installed the py27 so that we don't need to install again. See the link:
For windows: https://github.com/Microsoft/azure-pipelines-image-generation/blob/master/images/win/Vs2017-Server2016-Readme.md
For linux: https://github.com/Microsoft/azure-pipelines-image-generation/blob/master/images/linux/Ubuntu1604-README.md
For macos: https://github.com/Microsoft/azure-pipelines-image-generation/blob/master/images/macos/macos-Readme.md